src/api/java/appeng/api/util/WorldCoord.java

```diff
@@ -148,7 +148,7 @@ public WorldCoord copy() {
 
     @Override
     public int hashCode() {
-        return (this.y << 24) ^ this.x ^ this.z;
+        return this.y << 24 ^ this.x ^ this.z;
```

Comment:
this neither, should probably use Object.hashCode instead anyway

Reply:
Yes, I already consider it here.

Reply:
This is actually pointless to change. Or actually care about. The class is mostly an artifact when `BlockPos` did not exist, and is pretty much unused except for the sublass `DimensionalCoord`. Something we should take care of in 1.17 and just remove it completely.